### PR TITLE
tick: added tick flavour support for passthrough fileboxes

### PR DESCRIPTION
**Pull request by dolikdev**
Passthrough filebox directory name will include Hold flavour
attribute (.H) when BoxesWithFlavours is defined in fidogate.conf
and Tick flavour set to hold in routing.

Signed-off-by: Andriy Dzedolik <dz@dolik.dev>

### CODE REVIEW

**Comment by ykaliuta** [comment]
No problem, but was it enough to force update the branch instead of creating new PR?
I would prefer to move the change to a helper but for the current code it's ok as is.

I'll update the commit message a bit probably, hope it's ok for you.

Thank you for the contribution!

**Comment by ykaliuta** [comment]
Could you please review my version?

**Comment by dolikdev** [comment]
The 'if' condition indeed a bit hard to follow, but otherwise looks better :)

**Comment by ykaliuta** [comment]
Could you test it before I push? Since unfortunately I do not have such a configuration at the moment. It's "staging" branch.

**Comment by dolikdev** [comment]
Built FreeBSD package & deployed on the node. Will update you on results.

**Comment by dolikdev** [comment]
Verified, works as expected in all 3 modes:
- no PassthroughtBoxesDir key - legacy behavior;
- PassthroughtBoxesDir key + tick routing hold - adds ".H" to directory name;
- PassthroughtBoxesDir key + tick routing normal - no ".H" in the directory name;

**Comment by ykaliuta** [comment]
> Verified, works as expected in all 3 modes:
> 
>     * no PassthroughtBoxesDir key - legacy behavior;
> 
>     * PassthroughtBoxesDir key + tick routing hold - adds ".H" to directory name;
> 
>     * PassthroughtBoxesDir key + tick routing normal - no ".H" in the directory name;

Thank you!
